Content.Shared/Access/AccessLevelPrototype.cs

```diff
@@ -8,7 +8,7 @@ namespace Content.Shared.Access
     /// <summary>
     ///     Defines a single access level that can be stored on ID cards and checked for.
     /// </summary>
-    [Prototype("accessLevel")]
+    [Prototype("AccessLevel")]
```

Comment:
Prototypes are camelCase, don't rename this.

Reply:
Done

Reply:
You need to actually update all the prototypes you changed now to use the correct naming again.